TLM/TLM/Manager/Impl/LaneArrowManager.cs

```diff
@@ -111,6 +112,43 @@ public class LaneArrowManager
             return false;
         }
 
+        /// <summary>
+        /// Resets lane arrows to their default value for the given segment end.
+        /// </summary>
+        /// <param name="segmentId">segment to reset</param>
+        /// <param name="startNode">determines the segment end to reset. if <c>null</c>
+        /// both ends are reset</param>
+        public void ResetLaneArrows(ushort segmentId, bool? startNode = null) {
+            foreach (var lane in netService.GetSortedLanes(
```

Comment:
Not like it matters, but do you need sorted lanes for that? Or is there a cheaper way to get all lanes?